gdx-controllers-desktop/src/main/java/com/badlogic/gdx/controllers/desktop/support/JamepadController.java

```diff
@@ -18,6 +19,8 @@ public class JamepadController implements Controller {
     private static final IntMap<ControllerButton> CODE_TO_BUTTON = new IntMap<>(ControllerButton.values().length);
     private static final IntMap<ControllerAxis> CODE_TO_AXIS = new IntMap<>(ControllerAxis.values().length);
     private static final Logger logger = new Logger(JamepadController.class.getSimpleName());
+    private static final ControllerButton[] CONTROLLER_BUTTON_VALUES = ControllerButton.values();
```

Comment:
Please add a comment why this is done so people only reading the code know there were performance issues

Reply:
Yes, good idea, can do!